### ISSUE TITLE
NPE crash at org.jsoup.parser.HtmlTreeBuilderState$7.inBodyStartTag(HtmlTreeBuilderState.java:454)

### ISSUE THREAD

**Issue by ValleZ** [jhy/jsoup#1531]
java.lang.NullPointerException
        at org.jsoup.parser.HtmlTreeBuilderState$7.inBodyStartTag(HtmlTreeBuilderState.java:454)
        at org.jsoup.parser.HtmlTreeBuilderState$7.process(HtmlTreeBuilderState.java:282)
        at org.jsoup.parser.HtmlTreeBuilder.process(HtmlTreeBuilder.java:136)
        at org.jsoup.parser.TreeBuilder.runParser(TreeBuilder.java:66)
        at org.jsoup.parser.TreeBuilder.parse(TreeBuilder.java:47)
        at org.jsoup.parser.Parser.parse(Parser.java:107)
        at org.jsoup.Jsoup.parse(Jsoup.java:58)

while parsing attached html via:
```
  String input = new String(contentData, contentCharset);
            input = Normalizer.normalize(input, Normalizer.Form.NFC);
            Document document = Jsoup.parse(input);
```

[jsoupc.zip](https://github.com/jhy/jsoup/files/6412379/jsoupc.zip)

**Comment by ValleZ** [comment]
v1.13.1

**Comment by krystiangorecki** [comment]
To reproduce: 
```
Jsoup.parse("<isindex />");
```

**Comment by krystiangorecki** [comment]
Duplicate of #1404. Fix already exists and will be available in [release 1.14.1](https://github.com/jhy/jsoup/milestone/18).

**Comment by jhy** [comment]
Thanks
